vizro-core/examples/visual-vocabulary/assets/css/custom.css

```diff
@@ -92,20 +92,26 @@ img[src*="#chart-icon"] {
 }
 
 .anchor-container {
-  background: #060a17;
```

Comment:
@nadijagraca - there was previously no color set and the background was hard-coded. This normally prevents it from looking good on a dark and light theme. So if possible, always use the CSS variables at hand, such that it's suitable for both themes 👍